vector/src/main/res/values-fr/strings.xml

```diff
@@ -2462,7 +2463,7 @@
     <string name="invalid_qr_code_uri">Code QR invalide (URI invalide) !</string>
     <string name="cannot_dm_self">Impossible avec vous-même !</string>
     <string name="share_by_text">Partager par SMS</string>
-    <string name="phone_book_perform_lookup">Rechercher des contacts sur Matrix</string>
+    <string name="template_phone_book_perform_lookup">Rechercher des contacts sur ${app_name}</string>
```

Comment:
Can you please revert this change? the request #248 has been cancelled for the moment